src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -331,6 +283,50 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
       { emitCurrentValue: true, clearSignal: this._initCanceller.signal },
     );
 
+    /**
+     * Callback allowing to reload the current content.
+     * @param {number} deltaPosition - Position you want to seek to after
+     * reloading, as a delta in seconds from the last polled playing position.
+     * @param {number|undefined} minimumPosition - If set, minimum time bound
+     * in seconds after `deltaPosition` has been applied.
+     * @param {number|undefined} maximumPosition - If set, minimum time bound
+     * in seconds after `deltaPosition` has been applied.
+     */
+    const reloadMediaSource = (
+      deltaPosition: number,
+      minimumPosition: number | undefined,
+      maximumPosition: number | undefined,
+    ): void => {
+      const contentInfo = this._currentContentInfo;
+      if (contentInfo === null) {
+        log.warn("MTCI: Asked to reload when no content is loaded.");
+        return;
+      }
+      const lastObservation = playbackObserver.getReference().getValue();
+      const currentPosition = lastObservation.position.getWanted();
+      const isPaused =
+        contentInfo.initialPlayPerformed?.getValue() === true ||
+        contentInfo.autoPlay === undefined
+          ? lastObservation.paused
+          : !contentInfo.autoPlay;
```

Comment:
I'm not sure why the`isPaused` logic is different for multithread than for main_thread?
```js
      const isPaused =
        lastObservation.paused.pending ??
        coreObserver.getIsPaused() ??
        lastObservation.paused.last;
```

Reply:
The two are roughly equivalent, as `paused.pending` should only be different from `paused.last` when an auto-play is in waiting.

The monothread version is preferable because easier to read (either there is a pause pending in which case we take it, either we rely on what the video element tells us, either we rely on the last observation).

But I wrote it that way in the MultiThreadContentInitializer because though it could theoretically have access to the CorePlaybackObserver from which those other data are obtained, I found that it would have led to less maintainable code to store it in its state.